lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -1032,7 +1033,16 @@ sub save_edit :Chained('base') :PathPart('save') :Args(0) {
             if ($field =~ /designer|producer/){
                 return $c->forward($c->view('JSON')) unless ($complat_user_admin || $value eq '');
             } elsif ($field eq "maintainer") {
-                return $c->forward($c->view('JSON')) unless $value = format_maintainer($c, $value, $ia, 0);
+                my $result = format_maintainer($c, $value, $ia, 0);
+               use Data::Dumper;
+               print Dumper $result;
```

Comment:
Yeah you're right, it will never pass that if! I think I might have made some change at some point to make the maintainer undefined when there's no github account and then undid it, and forgot about the implications...

Fixing it!